src/main/java/com/graphhopper/converter/data/CountryInfo.java

```diff
@@ -0,0 +1,296 @@
+/*******************************************************************************
+ *   Gisgraphy Project 
+ * 
+ *   This library is free software; you can redistribute it and/or
+ *   modify it under the terms of the GNU Lesser General Public
+ *   License as published by the Free Software Foundation; either
+ *   version 2.1 of the License, or (at your option) any later version.
+ * 
+ *   This library is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ *   Lesser General Public License for more details.
+ * 
+ *   You should have received a copy of the GNU Lesser General Public
+ *   License along with this library; if not, write to the Free Software
+ *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA
+ * 
+ *  Copyright 2008  Gisgraphy project 
+ *  David Masclet <davidmasclet@gisgraphy.com>
+ *  
+ *  
+ *******************************************************************************/
+package com.graphhopper.converter.data;
+
+import java.io.File;
+import java.util.HashMap;
+
+/**
+ * 
+ * some information on country
+ * 
+ * @author <a href="mailto:david.masclet@gisgraphy.com">David Masclet</a>
+ * 
+ */
+public class CountryInfo {
```

Comment:
Regarding this class, AFAIK you want to add the country to Gisgraphy. Should we then even add this class, wait until the country is available in Gisgraphy, or use the country code for now and switch to the country name, once it's available in the Gisgraphy API?

Reply:
I suggest keeping this code because other geocoder returns the country name and not the iso-3166-2 one. The code can be done but needs time. The goal is to have a first PR that match as best as possible to ease the migration of users that want to use Gisgraphy. Then we will improve it step by step (as for locale parameter). Gisgraphy is not to be a perfect equivalent to other providers, even if I think the proposed changes make sens. Gisgraphy has some fields that are not present in Nominatim (formatedpostal, azimuthStart, azimuthEnd, length,...) and I am not sure they modify their feed to match Gisgraphy. :-). hopefully, there are some improvements in Gisgraphy that can be done ;-)

Reply:
WDYT @karussell, should we just use the country code or should we convert the country code into the actual country name?

Reply:
I'm good with what @gisgraphy said :+1:

Reply:
> offering country name via this converter or did I misunderstood sth :)?

No that's correct :). Ok, let's keep it as it is.